src/sentry/static/sentry/app/views/alerts/details/header.tsx

```diff
@@ -229,19 +233,13 @@ const IncidentsLink = styled(Link)`
   color: inherit;
 `;
 
-const Chevron = styled(InlineSvg)`
-  color: ${p => p.theme.gray1};
-  margin: 0 ${space(0.5)};
-`;
-
-const StyledMenuItem = styled(MenuItem)`
-  font-size: ${p => p.theme.fontSizeMedium};
-  text-align: left;
-  padding: ${space(1)} 12px; /* To match dropdown */
-  white-space: nowrap;
-`;
-
-const ResolveIcon = styled(InlineSvg)`
-  color: ${p => p.theme.greenLight};
-  margin-right: ${space(0.5)};
+const StatusMenuItem = styled(MenuItem)`
+  > span {
```

Comment:
Had to do this because the MenuItem wraps it. maybe a tad ugly but I think okay for now.